internal/events/events.go

```diff
@@ -0,0 +1,326 @@
+package events
```

Comment:
This still has Talos nomenclature in comments. Remove references to "runtime" and "v1alpha1."